src/compiler/parser/index.js

```diff
@@ -33,7 +33,7 @@ const dynamicArgRE = /^\[.*\]$/
 const argRE = /:(.*)$/
 export const bindRE = /^:|^\.|^v-bind:/
 const propBindRE = /^\./
-const modifierRE = /\.[^.\]]+(?=\.|$)/g
+const modifierRE = /(\.[^.\]]+)+$/g
```

Comment:
Use `/\.[^.\]]+(?=[^\]]*$)/g` instead so we can keep `parseModifiers` as is. I created a [benchmark](https://jsperf.com/vue-modifier-reg/) and it shows this is more performant.

Reply:
You are so amazing, thank you very much, I have modified the code according to your suggestion.